src/main/java/org/truffleruby/core/thread/ThreadManager.java

```diff
@@ -74,6 +75,8 @@
 
     private final ThreadLocal<UnblockingAction> blockingNativeCallUnblockingAction = ThreadLocal.withInitial(() -> EMPTY_UNBLOCKING_ACTION);
 
+    private TruffleObject pthread_self, pthread_kill;
```

Comment:
Style: camelCase?
Style: split into multiple declarations?

Reply:
it's named after the native function name, and I think `pthreadSelf` would be weird.
I can split, but they are closely related so it seemed a nice occasion for this usage. But you're right, it's inconsistent with the rest of the codebase.